infra/Pos/Arbitrary/Infra.hs

```diff
@@ -42,5 +42,5 @@ instance Arbitrary VerInfo where
     shrink = genericShrink
 
 instance HasConfiguration => Arbitrary (DataMsg ProxySKHeavy) where
-    arbitrary = genericArbitrary
-    shrink = genericShrink
+    arbitrary = DataMsg <$> arbitrary
+    shrink (DataMsg x) = map DataMsg $ shrink x
```

Comment:
Can you explain this change?

Reply:
This is legacy. Sorry, I'll see what I can clean up. This PR is 100 years old :(